packages/e2e-tests/fixtures/execution-details-page.js

```diff
@@ -0,0 +1,21 @@
+const { AuthenticatedPage } = require('./authenticated-page');
+const { expect } = require('@playwright/test');
+
+export class ExecutionDetailsPage extends AuthenticatedPage {
+  constructor(page) {
+    super(page);
+
+    this.executionCreatedAt = page.getByTestId('execution-created-at');
+    this.executionId = page.getByTestId('execution-id');
+    this.executionName = page.getByTestId('execution-name');
+    this.executionStep = page.getByTestId('execution-step');
+  }
+
+  async verifyExecutionData(flowId) {
+    await expect(this.executionCreatedAt).toContainText(/\d+ seconds? ago/);
+    await expect(this.executionId).toHaveText(
```

Comment:
This value should match the UUID from the URL. This might eliminate some false positive scenarios.